patterns/vsi-extension/README.md

```diff
@@ -1,27 +1,7 @@
 # Add a VSI to a landing zone VPC
 
-This logic creates a VSI to an existing landing zone VPC.
+![Architecture diagram for adding a VSI to your VPC landing zone deployable architecture](../../reference-architectures/vsi-extension.drawio.svg)
 
-This code creates and configures the following infrastructure:
-- Adds an SSH key to IBM Cloud or uses an existing one.
-- Adds a VSI in each subnet of the landing zone VPC.
+This architecture creates a VSI in some or all of the subnets of your existing landing zone VPC deployable architecture.
 
-There are two ways through which a user can pass the VPC details for deploying the VSI, both the approaches are mutually exclusive.
-
-## Using `vpc_id`
-
-The VPC ID of the landing zone VPC can be assigned to the variable vpc_id in order to create a VSI within that specific VPC.
-
-## Using `prerequisite_workspace_id` and `existing_vpc_name`
-
-The user can specify the workspace ID associated with the deployment of the landing zone VPC when creating a new VSI.
-
-Follow these steps to get the schematics workspace ID.
-
-1. Click the Navigation menu icon, and then click Schematics > Workspaces.
-1. Select the Workspace that is associated with landing zone VPC.
-1. Click the Settings.
-1. In the Details section, you can find the Workspace ID.
-
-Pass the Workspace ID to the `prerequisite_workspace_id` variable and pass the name of the VPC to the `existing_vpc_name` to choosse the name of the VPC to which the user wants to deploy the VSI.
-Please provide the Workspace ID for the prerequisite workspace and the name of the existing VPC to the `prerequisite_workspace_id` and `existing_vpc_name` variables respectively, to identify the VPC where you want to deploy the VSI.
+To deploy this architecture, you need the VPC ID, subnet names, and boot volume encryption key from your existing VPC landing zone deployable architecture. For information about finding the information, see [Adding a VSI to your VPC landing zone deployable architecture](https://test.cloud.ibm.com/docs-draft/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-ext-with-vsi).
```

Comment:
I think we need a "Before you begin" section that mentions the block storage to KMS auth policy must exist before provisioning this DA. And that it would of been created by the VPC landing zone DA if `add_kms_block_storage_s2s` was set to true which it is by default.

Reply:
that link is for IBM Cloud stage. remove the `test` from the URL and it will be a prod link